tensorflow_estimator/python/estimator/training.py

```diff
@@ -903,6 +903,10 @@ def _execute_evaluator_once(self, evaluator, continuous_eval_listener,
           'before finding (next) new checkpoint. Please consider to increase '
           'it.')
 
+    # Resets all state generated by Keras.
+    # If not do clear, it will lead to memory leak.
+    tf.keras.backend.clear_session()
```

Comment:
I am not sure if it will cause any issue since this function is executed by _start_continuous_evaluation, and might incur some performance loss if the session is cleared and some cache might lost. Also since I am not the expert for estimator, we might need someone else to review this change.

Reply:
I see, I will close this PR.